projects/ad738x-st/Makefile

```diff
@@ -0,0 +1,11 @@
+# Select the example you want to enable by chossing y for enabling and n for disabling
+IIO_EXAMPLE = y
+
+# Select the device you want to enable by choosing y for enabling and n for disabling
+include ../../tools/scripts/generic_variables.mk
+
+include src.mk
+
+include ../../tools/scripts/generic.mk
+
+CFLAGS += -O2
```

Comment:
any reason why this is added?

Reply:
better performance. but ill remove it, and will add it in another PR with performance improvements.

Reply:
in general, we keep the default -O0 seeing how these are project examples which are expected to be built, run and debugged on targets by our customers. 
if better performance is needed by a release (for a client) I would advice using RELEASE=y in builds.json which would generate a binary compiled with O2 which can be used to program the target directly.
see https://github.com/analogdevicesinc/no-OS/blob/master/tools/scripts/generic.mk#L169C1-L172C1